pino.js

```diff
@@ -58,7 +58,8 @@ const normalize = createArgsNormalizer(defaultOptions)
 const serializers = Object.assign(Object.create(null), stdSerializers)
 
 function pino (...args) {
-  const { opts, stream } = normalize(...args)
+  const instance = {}
+  const { opts, stream } = normalize.apply(instance, args)
```

Comment:
pass the instance into `createArgsNormalizer` instead of using `apply` here

Reply:
How could I pass instance to  `createArgsNormalizer` if it's declared in upper scope and I see no way how to pass the instance that's declared in `pino` function into function declared in upper scope.